pkg/url/url.go

```diff
@@ -601,7 +602,21 @@ func Push(client *occlient.Client, kClient *kclient.Client, parameters PushParam
 	// find URLs to delete
 	for urlName, urlSpec := range urlCLUSTER {
 		val, ok := urlLOCAL[urlName]
-		if !ok {
+
+		configMismatch := false
+		if ok {
+			// since the host stored in an ingress
+			// is the combination of name and host of the url
+			if val.Spec.urlKind == envinfo.INGRESS {
+				val.Spec.Host = fmt.Sprintf("%v.%v", urlName, val.Spec.Host)
```

Comment:
Can we get the host from `urlSpec.Spec.Host` rather than constructing it?

Reply:
One way would be a regrex operation but that would be slower and more lines of code